common/changes/@itwin/imodel-react-hooks/imodel-react-hooks-4_2023-09-29-15-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/imodel-react-hooks",
+      "comment": "update @itwin/imodel-react-hooks to support itwin.js@4",
+      "type": "patch"
```

Comment:
Maybe make it minor?

Reply:
Perhaps because of the peer minimum supported typescript version, otherwise there are no backwards incompatible changes afaict... I'll bump minor

Reply:
done